docs/projects/zenfs.md

```diff
@@ -3,10 +3,10 @@
 <a href="https://rocksdb.org/" target="_blank">*RocksDB*</a> is a persistent
 key-value store for fast storage devices. It is implemented using a
 <a href="https://en.wikipedia.org/wiki/Log-structured_merge-tree"target="_blank">
-Log-Structured Merge-Tree (LSM-tree)</a> data structure. Similarly to all
+Log-Structured Merge-Tree (LSM-tree)</a> data structure. Similar to all
```

Comment:
That is not correct. "similar" is an adjective while "similarly" is an adverb. Adjective can only modify nouns, while adverbs apply to clauses, which is what we have here ("to all LSM-tree..."). I removed this change from your patch.